reflex/components/next/image.py

```diff
@@ -104,6 +105,6 @@ def check_prop_type(prop_name, prop_value):
 
         src = props.get("src", None)
         if src is not None and not isinstance(src, (Var)):
-            props["src"] = Var.create(value=src, _var_is_string=True)
+            props["src"] = LiteralVar.create(src)
```

Comment:
another place where it seems like this would just be handled by `Component.__init__`